apps/docusaurus/static/scripts/install_cli.py

```diff
@@ -32,6 +32,14 @@
     with warnings.catch_warnings():
         warnings.simplefilter("ignore", category=DeprecationWarning)
         from distutils.version import StrictVersion as Version
+except ImportError:
+    sys.stderr.write(
+        colorize(
+            "warning",
+            "Couldn't find distutils or packaging. We cannot check the current version of the CLI, we will install unconditionally.",
```

Comment:
Install unconditionally, I think may be a little confusing.  How about this:
```suggestion
            "Couldn't find distutils or packaging. We cannot check the current version of the CLI. We will install the latest version.",
```